OptimizelySDK.Tests/ProjectConfigTest.cs

```diff
@@ -281,17 +281,17 @@ public void TestInit()
 
             Assert.IsTrue(TestData.CompareObjects(expectedVariationIdMap, Config.VariationIdMap));
 
-			// Check Variation returns correct variable usage
+			// Check Variation returns correct variable usage and featureEnabled value.
             var featureVariableUsageInstance = new List<FeatureVariableUsage>
             {
                 new FeatureVariableUsage{Id="155560", Value="F"},
                 new FeatureVariableUsage{Id="155561", Value="red"},
             };
 
-            var expectedVariationUsage = new Variation { Id = "122231", Key = "Fred", FeatureVariableUsageInstances = featureVariableUsageInstance};
-            var actualVariationUsage = Config.GetVariationFromKey("test_experiment_multivariate", "Fred");
+            var expectedVariation = new Variation { Id = "122231", Key = "Fred", FeatureVariableUsageInstances = featureVariableUsageInstance, FeatureEnabled = true };
+            var actualVariation = Config.GetVariationFromKey("test_experiment_multivariate", "Fred");
```

Comment:
Please add the method like
/// test isFeatureEnabled property
assert.IsTrue(variationWithFeatureEnabled.FeatureEnabled)